### PR TITLE
 Dependency Updates && AndroidX migration && Kotlin Rewrite

### PR DESCRIPTION
**Pull request by akwasiio**

### CODE REVIEW

**Comment by akwasiio** [comment]
@emmanuelkehinde Can please help me get the build to pass? I don't know what to do. I don't have any license issues locally so it's hard to debug this

**Comment by emmanuelkehinde** [comment]
@oppongjr Don't worry about the build. It needs some update. I would do that later. Thanks for bringing that up.

**Comment by akwasiio** [comment]
@emmanuelkehinde Don't mention

File: shutdown/src/main/java/com/emmanuelkehinde/shutdown/Shutdown.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        /**
+         * Static method to be called that handles closing of the app interactively
+         *
+         * @param context the Activity as a context
+         * @param message the message to be displayed
+         * @param timeout the time before it resets, whereby user is required to press back twice again
+         */
+
+
```
Comment (emmanuelkehinde) [inline shutdown/src/main/java/com/emmanuelkehinde/shutdown/Shutdown.kt]: @oppongjr Just a small one, let's clear the white spaces/lines here.

**Comment by emmanuelkehinde** [comment]
Aside from the one comment, we are good. I've tested it locally.

**Comment by akwasiio** [comment]
@emmanuelkehinde Done!

**Comment by emmanuelkehinde** [comment]
Awesome! Thanks

**Pull request merged**